rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/RDTDeviceSelectorSpinnerFactoryTest.java

```diff
@@ -85,6 +85,8 @@ public void testGetViewsFromJsonShouldCorrectlyBootStrapWidget() throws Exceptio
         materialSpinner.setSelection(2);
         Mockito.verify(formUtils).populateRDTDetailsConfirmationPage(widgetArgsArgumentCaptor.capture(), ArgumentMatchers.eq(DEVICE_2));
         verifyWidgetArgsMatch(widgetArgsArgumentCaptor.getValue());
+
+        materialSpinner.getOnItemSelectedListener().onNothingSelected(null);
```

Comment:
Could we verify that the expected behaviour happens when this is called?

Reply:
verification added.